guides/source/action_mailbox_basics.md

```diff
@@ -265,9 +276,15 @@ https://actionmailbox:PASSWORD@example.com/rails/action_mailbox/sendgrid/inbound
 
 NOTE: When configuring your SendGrid Inbound Parse webhook, be sure to check the box labeled **“Post the raw, full MIME message.”** Action Mailbox needs the raw MIME message to work.
 
-## Examples
+## Processing Incoming Email
 
-Configure basic routing:
+Processing incoming emails usually entails using the email content to create models, update views, queue background work, etc. in your Rails application.
+
+Before you can start processing incoming emails, you'll need to setup Action Mailbox routing and create mailboxes.
+
+### Configure Routing
+
+After an incoming email is received via the configured ingress, it needs to be forwarded to a mailbox for actual processing by your application. Much like the [Rails router](https://guides.rubyonrails.org/routing.html) that dispatches URLs to controllers, routing in Action Mailbox defines which emails go to which mailboxes for processing. Routes are added to the `application_mailbox.rb` file using regular expression:
```

Comment:
My bad, I think I previously referenced the full guide URL when I sent feedback 😅, but we can / should link to the file only for the same reasons I mentioned on another comment. (easier to test locally, etc.)

```suggestion
After an incoming email is received via the configured ingress, it needs to be forwarded to a mailbox for actual processing by your application. Much like the [Rails router](routing.html) that dispatches URLs to controllers, routing in Action Mailbox defines which emails go to which mailboxes for processing. Routes are added to the `application_mailbox.rb` file using regular expression:
```

Reply:
No worries. I *thought* I caught that but fixed now!